mmedit/datasets/base_matting_dataset.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, ann_file, pipeline, data_prefix=None, test_mode=False):
         self.data_prefix = str(data_prefix)
         self.data_infos = self.load_annotations()
 
-    def evaluate(self, results, logger=None):
+    def evaluate(self, results):
```

Comment:
Reason for this change?

Reply:
Because I have found that `logger` has not been used in this function.

Reply:
The caller might pass logger into